.travis.yml

```diff
@@ -14,9 +17,14 @@ before_script:
 
     # Workaround for Selenium #3280 issue
   - "sudo sed -i 's/^127\\.0\\.0\\.1.*$/127.0.0.1 localhost/' /etc/hosts"
+
+  - "cd node_modules/pouchdb && npm install && cd -"
+  - "rm -fr node_modules/pouchdb/node_modules/express-pouchdb"
+  - "ln -s ../../express-pouchdb node_modules/pouchdb/node_modules/express-pouchdb"
+
   - "cd node_modules/pouchdb-server && npm install && cd -"
   - "rm -fr node_modules/pouchdb-server/node_modules/express-pouchdb"
-  - "ln -s ../../.. node_modules/pouchdb-server/node_modules/express-pouchdb"
+  - "ln -s ../../express-pouchdb node_modules/pouchdb-server/node_modules/express-pouchdb"
```

Comment:
I'm guessing that the above 6 lines don't actually need to be changed, and that that's why it's failing in Travis? TBH I just messed around with this until it worked; it's a pretty hacky solution.